ui/apps/platform/src/services/ImageSbomService.ts

```diff
@@ -1,9 +1,20 @@
-import { saveFile } from './DownloadService';
+import FileSaver from 'file-saver';
+import { sanitizeFilename } from 'utils/fileUtils';
+
+import axios from './instance';
 
 export function generateAndSaveSbom({ imageName }: { imageName: string }): Promise<void> {
-    return saveFile({
-        method: 'post',
+    return axios({
+        method: 'POST',
         url: '/api/v1/images/sbom',
         data: { imageName },
+        timeout: 0,
+    }).then((response) => {
+        const fileName = sanitizeFilename(`${imageName}.sbom`);
```

Comment:
Currently the produced SBOMs are `.json` files - in the future when/if additional output types are supported (such as xml, csv, etc.) the extension may need to change - the backend does set the `Content-Disposition` header - thoughts on using that header for the filename?

https://github.com/stackrox/stackrox/blob/5635e147c52a02291579c0bb005e7699674e4be9/central/image/service/http_handler.go#L79

We could change the back-end to mimic the naming / santiziation as needed.

Reply:
Yeah this is a better idea, I'll go ahead and make the update and let the backend dictate the extension/filename. Then the UI will just accept the file without the need to parse headers, etc. and then update the behavior when more file types are supported.